lib/protocol/elementIdAttribute.js

```diff
@@ -19,7 +19,13 @@ let elementIdAttribute = function (id, attributeName) {
         throw new ProtocolError('number or type of arguments don\'t agree with elementIdAttribute protocol command')
     }
 
-    return this.requestHandler.create(`/session/:sessionId/element/${id}/attribute/${attributeName}`)
+    var command = '/attribute/'
+
+    if (this.options.useW3C && this.desiredCapabilities.browserName === 'firefox' && attributeName === 'value') {
```

Comment:
@APshenkin ok, let's at least have this check within a helper script so we only need to change the check in one file